roles/sap_ha_install_hana_hsr/tasks/main.yml

```diff
@@ -66,6 +66,14 @@
 
 # BEGIN of primary node connectivity test block
 - name: "SAP HSR - Check block to verify Ansible access to primary node"
+  # block settings
```

Comment:
I suggest to remove this line now.

Reply:
Removed in https://github.com/sap-linuxlab/community.sap_install/pull/259/commits/44b769a283bf2c73153f867d9114891255661b27